renovate.json

```diff
@@ -1,5 +1,32 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base"],
-  "enabledManagers": ["docker-compose","maven-wrapper","gradle-wrapper"]
+  "extends": [
+    "config:base",
+    ":separateMultipleMajorReleases",
+    ":automergeDigest",
+    "docker:pinDigests",
+    "docker:enableMajor"
+  ],
+  "enabledManagers": [
+    "docker-compose",
+    "maven-wrapper",
+    "gradle-wrapper"
+  ],
+  "packageRules": [
+    {
+      "matchDatasources": ["docker"],
+      "matchUpdateTypes": ["digest"],
+      "schedule": ["every weekend on the 1st through 7th day of the month"]
+    },
+    {
+      "matchPackageNames": ["postgres"],
+      "matchDatasources": ["docker"],
+      "allowedVersions": "< 17.0.0"
```

Comment:
So what exactly is the reason to not allow PostgreSQL 17 yet? Everything seems to work ok, at least in our local Docker Compose setup according to our integration tests (does not say everything but still, quite a lot).

Reply:
This is a question about how much we are going to follow the PodiumD releases.
My opinion is that we should follow this as close as we can, that means we have some more restrictions we have to add.

Reply:
My question though is: what does the PodiumD 2 release have to do with the version of PostgreSQL that we use in our Docker Compose file? Does it hurt if we are on the latest PostgreSQL version in our Docker Compose file? I would think not. This only becomes tricky if we ever use PostgreSQL specific constructs in ZAC I think. I would personally just use the latest PostgreSQL in our Docker Compose file. We also have our e2e tests to test our environment after all.

Reply:
Yes, we agreed to follow PodiumD and that is the scope of https://dimpact.atlassian.net/browse/PZ-4315. This PR is a first step/task of that story I guess and we should fix that.

Reply:
I'm fine with dropping the postgresql restriction as long as we're not relying on anything that is too version dependent. 

And indeed, this was intended to be extended with the PodiumD requirements